src/sketchup-stl/exporter.rb

```diff
@@ -325,34 +327,35 @@ def self.do_options
         #
         # Export and Cancel Buttons
         #
+        btn_cancel = SKUI::Button.new('Cancel') { |control|
+          control.window.close
+        }
+        btn_cancel.position(-10, -10)
+        window.add_control(btn_cancel)
+
         btn_export = SKUI::Button.new('Export') { |control|
           write_setting('export_units'   , OPTIONS['export_units'])
           write_setting('stl_format'     , OPTIONS['stl_format'])
           write_setting('selection_only' , OPTIONS['selection_only'])
           control.window.close
           export_entities = get_export_entities
           if export_entities
-             path = select_export_file
-             begin
-                export(path, export_entities, OPTIONS) unless path.nil?
-             rescue => exception
-                msg = "SketchUp STL Exporter:\n"
-                msg << "An error occured during export.\n\n"
-                msg << exception.message << "\n"
-                msg << exception.backtrace.join("\n")
-                UI.messagebox(msg, MB_MULTILINE)
-             end
+            path = select_export_file
+            begin
+              export(path, export_entities, OPTIONS) unless path.nil?
+            rescue => exception
+              msg = "SketchUp STL Exporter:\n"
+              msg << "An error occured during export.\n\n"
+              msg << exception.message << "\n"
+              msg << exception.backtrace.join("\n")
+              UI.messagebox(msg, MB_MULTILINE)
+            end
           end
         }
 
-        btn_export.position(125, -5)
+        btn_export.position(-btn_cancel.width - 25, -10)
         window.add_control(btn_export)
 
-        btn_cancel = SKUI::Button.new('Cancel') { |control|
-          control.window.close
-        }
-        btn_cancel.position(-5, -5)
-        window.add_control(btn_cancel)
 
         window.default_button = btn_export
```

Comment:
Are we setting the cancel button for the dialog?

http://www.rubydoc.info/github/thomthom/SKUI/SKUI/Window#cancel_button-instance_method